keps/sig-testing/5468-invariant-testing/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 5468
+authors:
+  - "@BenTheElder"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-instrumentation
+  - sig-api-machinery
+status: provisional
+creation-date: 2025-09-04
+reviewers:
```

Comment:
maybe @jpbetz ?

richa per https://github.com/kubernetes/enhancements/issues/5468#issuecomment-3222962160

patrick and antonio as fellow SIG Testing TLs